policyengine_us/variables/gov/states/sc/tax/income/deductions/net_capital_gains_deduction/sc_net_capital_gain_deduction.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class sc_net_capital_gains_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina net capital gains deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.SC
+    reference = (
+        "https://dor.sc.gov/forms-site/Forms/IITPacket_2021.pdf#page=15",
+        "https://www.scstatehouse.gov/code/t12c006.php"
+        # South Carolina Code of Laws Section 12-6-1150 (A)
+    )
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.sc.tax.income.deductions.net_capital_gains_deduction
+        person = tax_unit.members
+        capital_gains = person("capital_gains", period)
```

Comment:
```suggestion
        capital_gains = add(tax_unit, period ["capital_gains"])
```

Reply:
Hi Pavel, I tried to use the code you provided, but there's the error message: `TypeError: tuple indices must be integers or slices, not str` How should I fix this?

Reply:
there is a comma missing after "period" - if have edited the comment